amundsen_application/jira/jira_client.py

```diff
@@ -0,0 +1,78 @@
+from typing import Any
+from jira import JIRA, JIRAError, Issue
+from flask import current_app as app
+
+import logging
+
+SEARCH_STUB = 'project={project_id} AND text ~ "{table_key}"'
+
+
+class JiraClient:
+
+    def __init__(self) -> None:
```

Comment:
How about taking values from app from constructor so that we can have this client decouple from `flask`?